### ISSUE TITLE
What is exactly breaking in v29?

### ISSUE THREAD

**Issue by yvele** [google/google-api-nodejs-client#1146]
29 is a new major version, but I can't really see what exactly is breaking https://github.com/google/google-api-nodejs-client/releases/tag/v29.0.0 🤔 

Is it possible to add a synthetic breaking node to the changelog?

PS: By the way, nice lib

**Comment by JustinBeckwith** [comment]
Greetings!  In all likelihood...  nothing.  The majority of this library is auto-generated. We tend to do a major version bump when we run the generator between releases, as we can't make a 100% guarantee that nothing significant changed. The change that included a run of the generator is here:
https://github.com/google/google-api-nodejs-client/pull/1108/files

I dug around - and to be honest, it doesn't look like anything was semver major.  In the interest of being overly cautious though...  we decided to go to 29.0. 

We're going to take a deeper look at automating this process in #525, which hopefully will make this a little more fool proof for me :)

TL;DR - you're probably fine :)

**Comment by JustinBeckwith** [comment]
Side note:

> Is it possible to add a synthetic breaking node to the changelog?

I'm not sure what this means.  Could you elaborate?

> PS: By the way, nice lib

Thank you 🤗

**Comment by yvele** [comment]
Hum.. 🤔  Ok thanks 😉

**Comment by yvele** [comment]
> Is it possible to add a synthetic breaking node to the changelog?

I mean explaining, in the [GitHub release note](https://github.com/google/google-api-nodejs-client/releases/tag/v29.0.0), what is the reason for a major (breaking) version.. Developers may want to know if updating the package needs some refactoring, etc..

**Comment by JustinBeckwith** [comment]
Of course! We'll be a little more crisp about that next time :)
